### ISSUE TITLE
About the F function used to compute the gradient w.r.t. input?

### ISSUE THREAD

**Issue by wangyongjie-ntu** [pytorch/captum#418]
It seems the integrated gradient, deeplift, etc. compute the gradient of **logits** w.r.t input. 

Why not use the probability after softmax?  Are there any differences between them?  

Thanks

**Comment by wangyongjie-ntu** [comment]
please refer to the [issue 407](https://github.com/pytorch/captum/issues/407.)
